pkg/executables/clusterctl.go

```diff
@@ -26,6 +29,7 @@ const (
 	etcdadmBootstrapProviderName  = "etcdadm-bootstrap"
 	etcdadmControllerProviderName = "etcdadm-controller"
 	kubeadmBootstrapProviderName  = "kubeadm"
+	clusterctlMoveTimeout         = 30 * time.Minute
```

Comment:
could you add a comment as to where this came from?

Reply:
Added the requested comment.  This timeout is arbitrary.  I picked the value established by EKS-A for kubectl waits.  This duration really defines how long EKS-A will wait for a transient network disruption that occurs during a move to abate.  I'm open to suggestion about other durations.